src/pushsource/_impl/model/ami.py

```diff
@@ -71,3 +71,39 @@ class AmiPushItem(VMIPushItem):
         validator=optional(instance_of(AmiBillingCodes)),
     )
     """Billing codes associated with this image."""
+
+    image_name = attr.ib(type=str, default=None, validator=optional_str)
+    """image name  """
+
+    description = attr.ib(type=str, default=None, validator=optional_str)
+    """A description for the image"""
+
+    arch = attr.ib(type=str, default=None, validator=optional_str)
+    """Architecture type of the image"""
+
+    virt_type = attr.ib(type=str, default=None, validator=optional_str)
+    """Virtualization type """
+
+    accounts = attr.ib(type=str, default=None, validator=optional_str)
+    """Accounts """
+
+    groups = attr.ib(type=str, default=None, validator=optional_str)
+    """Groups"""
+
+    release_notes = attr.ib(type=str, default=None, validator=optional_str)
+    """"Release notes for AMI """
+
+    operating_system_name = attr.ib(type=str, default=None, validator=optional_str)
+    """Name of operating system"""
+
+    operating_system_version = attr.ib(type=str, default=None, validator=optional_str)
+    """Version of operating system"""
```

Comment:
Can't we use the [AmiRelease.version](https://github.com/release-engineering/pushsource/blob/master/src/pushsource/_impl/model/vms.py#L26-L28) for this?

Reply:
Ditto for this.